docs/getting-started.md

```diff
@@ -30,9 +30,9 @@ First, here are the things you need before starting:
   * USB Ethernet (wired) dongle
   * Bluetooth
 
-  For Raspberry Pi you will need to use the wired Ethernet connection first.
-  With PiStorms the EV3 options will work too since you have a screen to configure
-  them on.
+  For RasberryPi you can use wired Ethernet connection first.
+  With PiStorms the EV3 options will work too since you have a built-in screen to configure them on.
+  With BrickPi you can also setup other connection from the console using external screen and keyboard.
```

Comment:
I can change it but we use setup as verb all over the page so I would than change it all over.

You can setup additional connections using the
`connmanctl` command.

 Step 5: Setup a network connection

If that's desired.